comfy_cli/cql/engine.py

```
@@ -2156,7 +2198,9 @@ def _dynamic_combo_options(spec: Any) -> list[dict]:
     return [o for o in (options_meta.get("options") or []) if isinstance(o, dict)]
 
 
-def _check_dynamic_combos(node_id: str, class_type: str, m: Morphism, node_data: dict) -> tuple[list[dict], list[dict]]:
+def _check_dynamic_combos(
```

Pylint (4.0.7)

[refactor] Too many local variables (23/15)

(R0914)